tests/data/test_instance_cropping.py

```diff
@@ -0,0 +1,36 @@
+from sleap_nn.data.providers import LabelsReader
+import torch
+from sleap_nn.data.instance_cropping import make_centered_bboxes, InstanceCropper
+from sleap_nn.data.instance_centroids import InstanceCentroidFinder
+from sleap_nn.data.normalization import Normalizer
+
+
+def test_instance_cropper(minimal_instance):
+    datapipe = LabelsReader.from_filename(minimal_instance)
+    datapipe = InstanceCentroidFinder(datapipe)
+    datapipe = Normalizer(datapipe)
+    datapipe = InstanceCropper(datapipe, 100, 100)
+    sample = next(iter(datapipe))
+
+    # test bounding box calculation
+    gt = torch.Tensor(
+        [
+            [72.4970, 130.5748],
+            [172.4970, 130.5748],
+            [172.4970, 230.5748],
+            [72.4970, 230.5748],
+        ]
+    ).int()
+    bbox = make_centered_bboxes(sample["centroids"], 100, 100).int()
+    assert torch.equal(gt, bbox)
```

Comment:
Definitely don't check for correctness by converting to integers!! This is losing precision and there's no reason to do it in this context.